packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -923,7 +924,22 @@ class _InteractiveViewerState extends State<InteractiveViewer> with TickerProvid
         _transformationController.value,
         focalPointSceneScaled - focalPointScene,
       );
+      final double scale = _transformationController.value.getMaxScaleOnAxis();
+      if (widget.onInteractionStart != null) {
+        widget.onInteractionStart(
+            ScaleStartDetails(focalPoint: focalPointSceneScaled));
+      }
+      if (widget.onInteractionUpdate != null) {
+        widget.onInteractionUpdate(ScaleUpdateDetails(
+          rotation: 0,
+          scale: scale,
+        ));
+      }
+      if (widget.onInteractionEnd != null) {
+        widget.onInteractionEnd(ScaleEndDetails());
+      }
```

Comment:
So for every mouse wheel event, we call all 3 callbacks: Start, Update, and End.  I guess that makes sense.  I don't think it's possible to group calls to _receivedPointerSignal.